src/components/SummaryPanel/InOutRateTable.tsx

```diff
@@ -13,7 +13,7 @@ type InOutRateTablePropType = {
   outRate5xx: number;
 };
 
-export class InOutRateTable extends React.Component<InOutRateTablePropType, {}> {
+export default class InOutRateTable extends React.Component<InOutRateTablePropType, {}> {
```

Comment:
I **do** like this for most use cases because it's nice and concise. 

However, when you decorate a class with a high order function the other method is needed:

`const FilterLink = connect(
  mapStateToProps,
  mapDispatchToProps
)(Link)

export default FilterLink`

And with _Redux_ now we will be doing more High Order functions like _connect_.

Reply:
Not sure what this means, I'll follow up with you tomorrow...